### PR TITLE
Feature/map 121/divide components

### PR DESCRIPTION
**Pull request by Yojuks**

### CODE REVIEW

File: frontend/src/components/Login/Login.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```
Comment (steadyGuy) [inline frontend/src/components/Login/Login.tsx]: ?

File: frontend/src/components/Login/Login.tsx [hunk 2]
```diff
@@ -9,6 +10,7 @@ import {
   SubmitHandler,
   useFormState
 } from 'react-hook-form';
+// import { useForm, SubmitHandler, useFormState } from 'react-hook-form';
```
Comment (steadyGuy) [inline frontend/src/components/Login/Login.tsx]: ?

**Review by steadyGuy — APPROVED** [review]

File: frontend/src/components/Login/LoginControllers/LoginControllers.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            onBlur={field.onBlur}
+            defaultValue={field.value}
+            error={!!errors.password?.message}
+            helperText={errors.password?.message}
+          />
+        )}
+      />
+    </>
+  );
+};
```
Comment (mazursasha1990) [inline frontend/src/components/Login/LoginControllers/LoginControllers.tsx]: Why you didn't remove this file?

File: frontend/src/components/Registration/Controllers/Controllers.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+              defaultValue={field.value}
+              error={!!errors.password?.message}
+              helperText={errors.password?.message}
+            />
+          )}
+        />
+      </Box>
+    </>
+  );
+};
```
Comment (mazursasha1990) [inline frontend/src/components/Registration/Controllers/Controllers.tsx]: Why you didn't remove this file?

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Login/Login.tsx (hunk 2 above)
Comment (Yojuks) [reply frontend/src/components/Login/Login.tsx]: fixed

File: frontend/src/components/Registration/Controllers/Controllers.tsx (hunk 4 above)
Comment (Yojuks) [reply frontend/src/components/Registration/Controllers/Controllers.tsx]: done

File: frontend/src/components/Login/LoginControllers/LoginControllers.tsx (hunk 3 above)
Comment (Yojuks) [reply frontend/src/components/Login/LoginControllers/LoginControllers.tsx]: done

File: frontend/src/components/Login/Login.tsx (hunk 1 above)
Comment (Yojuks) [reply frontend/src/components/Login/Login.tsx]: done

**Review by naberalgamma — APPROVED** [review]

File: frontend/src/components/Registration/Controllers/Controllers.tsx (hunk 4 above)
Comment (Yojuks) [reply frontend/src/components/Registration/Controllers/Controllers.tsx]: done

File: frontend/src/components/Map/ZoomComponent/index.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  const map = useMapEvents({
+    zoom: e => {
+      if (!isAddLocationActive) {
+        setBounds({ ...prev, ...map.getBounds() });
+      }
+    },
+    dragend: e => {
+      if (!isAddLocationActive) {
+        setBounds({ ...prev, ...map.getBounds() });
+      }
```
Comment (mazursasha1990) [inline frontend/src/components/Map/ZoomComponent/index.tsx:31]: Screenshot from 2022-06-02 10-14-52

File: frontend/src/components/Profile/EditGroup/EditGroup.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
+import { Box, Typography } from '@mui/material';
+import React, { FC } from 'react';
+import { useTypedSelector } from 'redux/hooks/useTypedSelector';
+import { useTranslation } from 'react-i18next';
+import userImageNotFound from '../../../static/image-not-found.jpg';
+import { EditButton, ProfileAvatar, UploadBox } from '../styles';
```
Comment (mazursasha1990) [inline frontend/src/components/Profile/EditGroup/EditGroup.tsx:6]: Screenshot from 2022-06-02 10-20-13

File: frontend/src/components/Profile/SaveGroup/SaveGroup.tsx [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+import { useTranslation } from 'react-i18next';
+import { UserForm } from '../../../../types';
+import {
+  CancelButton,
+  ProfileAvatar,
+  SaveBox,
+  SaveButton,
+  UploadBox
+} from '../styles';
+import UploadInputProfilePage from 'components/design/UploadInputProfilePage';
```
Comment (mazursasha1990) [inline frontend/src/components/Profile/SaveGroup/SaveGroup.tsx:20]: Screenshot from 2022-06-02 10-22-08

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Profile/EditGroup/EditGroup.tsx (hunk 6 above)
Comment (Yojuks) [reply frontend/src/components/Profile/EditGroup/EditGroup.tsx:6]: done

File: frontend/src/components/Map/ZoomComponent/index.tsx (hunk 5 above)
Comment (Yojuks) [reply frontend/src/components/Map/ZoomComponent/index.tsx:31]: done

File: frontend/src/components/Profile/SaveGroup/SaveGroup.tsx (hunk 7 above)
Comment (Yojuks) [reply frontend/src/components/Profile/SaveGroup/SaveGroup.tsx:20]: done

**Review by mazursasha1990 — APPROVED** [review]

**Pull request merged**